docs/howtos/telemetry/20-tracing-qs.md

```diff
@@ -118,12 +118,13 @@ following contents:
 
 ```yaml
 telemetry:
-  tracing:
-    enabled: True
-    jaeger:
-      endpoint: "my-open-telemetry-collector.jaeger.svc.cluster.local:4317"
-      tls:
-        insecure: true
+  tracing: True
```

Comment:
`mode: sidecar` is not required here?

Reply:
sidecar mode is the default one. Therefore, it's not strictly required. But I agree that best to explicitly mention it. I'll update the docs